python/ray/serve/_private/router.py

```diff
@@ -37,6 +37,9 @@ class RequestMetadata:
     # and it needs to be deserialized by the replica.
     http_arg_is_pickled: bool = False
 
+    # HTTP route path of the request.
+    route: str = "N/A"
```

Comment:
Can you make `"N/A"` a constant in [`constants.py`](https://github.com/ray-project/ray/blob/b74253861a1c960613aa19b16d999a45ad08c605/python/ray/serve/_private/constants.py) and use the constant instead? This will help keep the value consistent across all the different files.

Reply:
should this be empty instead of `N/A`? what's the consideration here

Reply:
When user uses `handle` to trigger request, there won't be route & request id information. `None` is not acceptable type for ray core metrics.

Reply:
What about `""` or `"HANDLE"` (to match the logs)

Reply:
Go for the "".